Cargo.toml

```diff
@@ -10,7 +10,7 @@ readme = "README.md"
 publish = false
 default-run = "vector"
 autobenches = false # our benchmarks are not runnable on their own either way
-rust-version = "1.62.0"
+rust-version = "1.62.1"
```

Comment:
We don't want to bump this since it is the MSRV, rather than the Rust toolchain version.